### PR TITLE
Profile picture upload

### PR DESCRIPTION
**Pull request by nilscox**

### CODE REVIEW

File: .github/workflows/preview-env.yml [hunk 1]
```diff
@@ -68,9 +68,6 @@ jobs:
           )
           IFS=, ; echo "env=${variables[*]}" >> $GITHUB_OUTPUT
 
-      - name: Create files volume
```
Comment (Bopzor) [inline .github/workflows/preview-env.yml:71]: **question**

why it is not needed anymore?

File: apps/web/src/intl/lang/fr.json [hunk 2]
```diff
@@ -387,7 +387,8 @@
       "phoneNumberInvalid": "Ce numéro de téléphone n'est pas valide.",
       "bio": "Présentation",
       "profilePicture": "Photo de profil",
-      "profilePictureMessage": "La gestion des photos de profil n'est pas encore disponible directement via l'app. En attendant, il est possible de changer son image via <link>gravatar</link> (après inscription).",
+      "profilePictureMessage": "Pour changer votre photos de profil, cliquez sur le bouton ci-dessous et sélectionnez un fichier.",
```
Comment (Bopzor) [inline apps/web/src/intl/lang/fr.json:390]: **type**

`photo` should be singular

File: apps/web/src/modules/profile/profile-edition/profile-edition-form.tsx [hunk 3]
```diff
@@ -90,22 +96,51 @@ function BioField(props: Pick<ProfileEditionForm, 'errors'>) {
...
+
+        <div class="col gap-2">
+          <p>
+            <T id="profilePictureMessage" />
+          </p>
+
+          <input
+            ref={input}
+            type="file"
+            accept=".bmp,.gif,.jpg,.jpeg,.png"
```
Comment (Bopzor) [inline apps/web/src/modules/profile/profile-edition/profile-edition-form.tsx:135]: **praised**

git :heart_eyes:

File: apps/web/src/intl/lang/fr.json (hunk 2 above)
Comment (nilscox) [reply apps/web/src/intl/lang/fr.json:390]: Good catch

File: .github/workflows/preview-env.yml (hunk 1 above)
Comment (nilscox) [reply .github/workflows/preview-env.yml:71]: We need a simpler way to handle volumes for preview environments, I'm just simplifying the flow for now

**Pull request merged**